routing/control_tower.go

```diff
@@ -57,6 +57,11 @@ type ControlTower interface {
 	// sent out immediately.
 	SubscribePayment(paymentHash lntypes.Hash) (*ControlTowerSubscriber,
 		error)
+
```

Comment:
Nit about commit message: generally the sentence after the subsystem indicator isn't capitalized.

Reply:
Right, didn;t notice that, updated!